.gitignore

```diff
@@ -80,3 +80,4 @@ keys/*
 ####
 hemtt.exe
 ArmaScriptCompiler.exe
+.hemtt/project.toml
```

Comment:
delete pls, i know its annoying to remove content from this file before creating PR, but i dont want to limit it

Reply:
shouldn't that be on gitnore???

Reply:
why it should be? its basically basic config for hemtt